requirements.txt

```diff
@@ -1,16 +1,12 @@
-flask
```

Comment:
Since this is all being refactored, should we move to `pipenv`?

Reply:
That depends if we want to leverage snaps or not. Snapcraft only supports `requirements.txt`. However we could put a CI step to have `pipenv` dump a `requirements.txt` if we wanted to get the best of both worlds.

Reply:
In other good news, Python 3.6 is the only supported Python3 version in snapcraft, so we are good there!

Reply:
Based on what ductape said, let's stick with requirements.txt for now.